war/src/main/js/templates/pluginSelectList.hbs

```diff
@@ -4,21 +4,22 @@
   {{#each this}}
   <div class="plugin {{id plugin.name}} {{#inSelectedPlugins plugin.name}}selected{{/inSelectedPlugins}} {{#ifVisibleDependency plugin.name}}show-dependencies{{/ifVisibleDependency}}" id="row-{{plugin.name}}">
     {{#hasDependencies plugin.name}}
-      <div class="btn btn-link toggle-dependency-list" type="button" data-plugin-name="{{../plugin.name}}">
-        {{../../../translations.installWizard_installIncomplete_dependenciesLabel}}<span class="badge">{{dependencyCount ../plugin.name}}</span>
-      </div>
+      <a href="javascript:return false" class="btn btn-link toggle-dependency-list" type="button" data-plugin-name="{{../plugin.name}}" title="{{../plugin.title}} {{../../../translations.installWizard_installIncomplete_dependenciesLabel}}">
```

Comment:
@recena to match the change in the keyboard navigation PR, since I'm going to have to merge these together